src/test/file/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+find_package(GLIB REQUIRED)
```

Comment:
I'm pretty sure we already search for the GLIB package in `src/main/CmakeLists.txt`, so adding this here will cause us to search twice (and the GLib paths will be printed twice) when we build the tests. I think this is still better than moving this search to the toplevel `CMakeLists.txt` (so that the result can be inherited everywhere) and then removing the `find_package()` here and in `src/main/CmakeLists.txt`. Do you agree?

Reply:
Yup, good idea

Reply:
Heh, I went to move the search to the top-level, had second thoughts, and realized I'd misread your comment.

Unless we have reason to think the package search has some significant cost, I'm leaning slightly towards keeping it 'locally' in the places where we're actually using GLib. Since it sounds like you're also leaning that way, I'll leave it.

Reply:
👍